src/features/student/views/StudentSkillView/StudentSkillView.test.ts

```diff
@@ -33,5 +55,12 @@ BddTest().given('a student skill view', () => {
         currentBreadcrumbLink
       ])
     })
+    BddTest().then('it should render StudentSkillViewContainer with correct props', async () => {
```

Comment:
```suggestion
    })
    
    BddTest().then('it should render StudentSkillViewContainer with correct props', async () => {
```

Là pour le coup on peut aérer un peu ;)